src/flint/fmpz.jl

```diff
@@ -1744,6 +1744,20 @@ function rand_bits(::FlintIntegerRing, b::Int)
    return z
 end
 
+@doc Markdown.doc"""
+    rand_prime(::FlintIntegerRing, n::Int, proved::Bool=true)
+> Returns a random prime number with the given number of bits. If only a
+> probable prime is required, one can pass `proved=false`.
+"""
+function rand_prime(::FlintIntegerRing, n::Int, proved::Bool = true,
```

Comment:
I agree it would probably make more sense to be a positional argument. But let's decide if we want to make that change everywhere. That should probably be discussed separately, as it affects other code (including Hecke).